services/autoscaling/src/simcore_service_autoscaling/models.py

```diff
@@ -37,37 +62,13 @@ def __sub__(self, other: "Resources") -> "Resources":
         )
 
 
-class EC2Instance(BaseModel, frozen=True):
+@dataclass(frozen=True)
+class EC2InstanceType:
```

Comment:
So this is information of the ec2 instance (even before it is started)  and EC2InstaceData is extra/other information of the instance once is started, right?

Reply:
yes